resolwe_bio/tools/lofreq2_indel_ovlp.py

```diff
@@ -90,10 +90,11 @@ def main():
                 prev_vars.append(cur_var)
 
             # don't forget remaining ones
-            picked_var = sorted(
-                prev_vars, key=lambda e: pick_best_func(e), reverse=True
-            )[0]
-            write_var(picked_var, out_vcf)
+            if prev_vars:
```

Comment:
If given an empty VCF file to this script (header does not count), `prev_vars` list variable would always be empty, and the script would fail with:
```
Traceback (most recent call last):
  File "/usr/local/bin/resolwe/4/lofreq2_indel_ovlp.py", line 94, in <module>
    main()
  File "/usr/local/bin/resolwe/4/lofreq2_indel_ovlp.py", line 89, in main
    picked_var = sorted(prev_vars, key=lambda e: pick_best_func(e), reverse=True)[0]
IndexError: list index out of range
```